### PR TITLE
Change to batctl-call to avoid debugfs

### PR DESCRIPTION
**Pull request by tackin**

### CODE REVIEW

File: providers/neighbours/batadv.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def call(self, batadv_dev):
                 glob('/sys/class/net/{}/lower_*'.format(batadv_dev))
         })([
             (line[0], float(line[1].strip('s')), int(line[2].strip(')')), line[4].strip('[]:'))
-            for line in map(lambda l: l.replace('(', '').replace('[', '').split(),
-                open('/sys/kernel/debug/batman_adv/{}/originators'.format(batadv_dev)))
+            for line in map(lambda l: l.replace('* ', '').replace('(', '').replace('[', '').split(),
+                 call(['batctl', '-m', batadv_dev, 'originators', '-H', '-n']))
```
Comment (jplitza) [inline providers/neighbours/batadv.py]: You've got one space too many:
```suggestion
                call(['batctl', '-m', batadv_dev, 'originators', '-H', '-n']))
```

**Comment by jplitza** [comment]
Please revert the last commit, it breaks compatibility with batman-adv 2013.4

**Comment by tackin** [comment]
Should we still really care for a 2013.x-version in this case?

**Comment by jplitza** [comment]
Well, as we are using it, I do care.

**Comment by jplitza** [comment]
Well, after #41 was merged without any discussion, this seems to be obsolete. Sorry.

**Comment by TobleMiner** [comment]
Oops, sorry.

People were complaining about the need for enabled debugfs in the mesh-announce Matrix channel for some time. Thus it didn't cross my mind that there might already be a PR for it.

**Comment by tackin** [comment]
Ok, but sorry to say that #41 will not work always as expected.

**Comment by TobleMiner** [comment]
@tackin Oh, I must have missed that in testing. I got confirmation that the current master branch works just fine though. Have you tried with the latest commits applied? Please let me know in case there are still problems with the current master branch.

**Pull request closed without merging**